### PR TITLE
Feat/add version 8 to 9 converter

### PR DESCRIPTION
**Pull request by TheGreatRefrigerator**

### CODE REVIEW

**Comment by TheGreatRefrigerator** [comment]
Currently, graphs_root_path is just removed without further info.
The final output should contain a bit more of what's new and where to find docu for it

**Comment by jhaeu** [comment]
From 8 -> 9 please migrate 

`ors.engine.graphs_root_path` -> `ors.engine.profile_default.graph_path`

**Review by jhaeu — CHANGES_REQUESTED** [review]

File: migrate.py [hunk 1]
```diff
@@ -412,6 +415,131 @@ def migrate(json_config_path, yaml_config_path):
...
+                if_exists_move_to(x, f'ors.{level_string}.profiles.{profile}.{prop}',
+                                  f'ors.{level_string}.profiles.{profile_new}.service.{prop}')
+
+            if profile_new != profile:
+                remove_and_output(x, f'ors.{level_string}.profiles.{profile}', results)
+
+        if_exists_move_to(x, f'ors.{level_string}.source_file',
+                          f'ors.{level_string}.profile_default.build.source_file')
+
+        if_exists_move_to(x, f'ors.{level_string}.graphs_root_path', f'ors.{level_string}.profile_default.graphs_root_path')
```
[the remark this answers is not in the document]
Comment (merydian) [reply migrate.py:506]: This is the only occurrence

[the remark this answers is not in the document]
Comment (jhaeu) [reply migrate.py]: Sure? Is there no model for v9 that needs to be adapted?

**Comment by jhaeu** [comment]
I found out that when migrating v7 -> v8 `graphs_root_path` gets lost. This seems to be an old issue.

This peace of information should be located in

* v7 `ors.services.routing.profiles.default_params.graphs_root_path`
* v8 `ors.engine.graphs_root_path`
* v9 `ors.engine.profile_default.graph_path`

**Comment by TheGreatRefrigerator** [comment]
restructured and fixed missing things in 9 allow extras model

**Pull request merged**